.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "@babel/preset-react"
+  ]
+}
```

Comment:
Use https://github.com/CruGlobal/give-web/blob/master/babel.config.js instead. It's the newer config file. Idk what having both will do...